websites/Y/YouTube/presence.ts

```diff
@@ -327,6 +327,8 @@ presence.on("UpdateData", async () => {
 					).textContent;
 					// Get channel name when viewing a channel
 				} else if (
+					document.querySelector("#text.ytd-channel-name")?.textContent !==
+						"" &&
```

Comment:
[Empty string are false.](https://stackoverflow.com/a/8693015/17773757)
```suggestion
					!document.querySelector("#text.ytd-channel-name")?.textContent &&
```